docker-compose-fabric.yaml

```diff
@@ -0,0 +1,114 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  tcf:
+    container_name: tcf
+    image: tcf-dev
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.tcf-dev
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - MAKECLEAN=${MAKECLEAN:-1}
+        - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
+        - DISPLAY=${DISPLAY:-}
+        - XAUTHORITY=~/.Xauthority
+    volumes:
+      # Below volume mappings are required for DISPLAY settings by heart disease GUI client
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
+      - ~/mywork/vars/keyfiles:/keyfiles
+      - ~/mywork/vars/mychannel_network.json:/project/TrustedComputeFramework/sdk/avalon_sdk/fabric/network.json
+    working_dir: "/project/TrustedComputeFramework"
+    command: |
+      bash -c "
+        source scripts/tcs_startup.sh -y -l http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+    stop_signal: SIGKILL
+    depends_on:
+      - avalon-lmdb
+
+  avalon-lmdb:
+    container_name: avalon-lmdb
+    image: avalon-lmdb-dev
+    build:
+      context: .
+      dockerfile: ./shared_kv_storage/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    expose:
+      # Port is where lmdb server will listen for http request.
+      # Port should be same as in bind parameter or lmdb_config.toml
+      - 9090
+    command: |
+      bash -c "
+        kv_storage --bind http://avalon-lmdb:9090
+        tail -f /dev/null
+      "
+
+  avalon-listener:
+    container_name: avalon-listener
+    image: avalon-listener-dev
+    build:
+      context: .
+      dockerfile: ./listener/Dockerfile
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    expose:
+      - 1947
+    command: |
+      bash -c "
+        avalon_listener --bind http://avalon-listener:1947 --lmdb_url http://avalon-lmdb:9090
+        tail -f /dev/null
+        "
+    depends_on:
+      - avalon-lmdb
+
+  avalon-blockchain-connector:
+    container_name: avalon-blockchain-connector
+    image: avalon-blockchain-connector-dev
+    build:
+      context: .
+      dockerfile: ./blockchain_connector/Dockerfile-fabric
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    command: |
+      bash -c "
+        avalon_blockchain_connector -b fabric -u http://avalon-listener:1947
+        tail -f /dev/null
+        "
+    volumes:
+      - ./:/project/avalon
+      - ~/mywork/vars/keyfiles:/keyfiles
+      - ~/mywork/vars/mychannel_network.json:/project/TrustedComputeFramework/sdk/avalon_sdk/fabric/network.json
+    depends_on:
+      - avalon-listener
+      - tcf
```

Comment:
Just FYI - After enclave manager PR is merged, tcf container just consists the avalon shell. So we need to change dependency of avalon-blockchain-connector from tcf to enclave manager.